knative-operator/deploy/resources/knativekafka/kafkachannel-v0.17.1.yaml

```diff
@@ -564,7 +623,7 @@ spec:
       containers:
       - name: kafka-webhook
         terminationMessagePolicy: FallbackToLogsOnError
-        image: gcr.io/knative-releases/knative.dev/eventing-contrib/kafka/channel/cmd/webhook@sha256:bcaa442b69619c073e4e6b71a5a14a07e2b652bc8ba59002de4aff8d4b21472a
+        image: registry.svc.ci.openshift.org/openshift/knative-v0.17.1:knative-eventing-sources-kafka-channel-webhook
```

Comment:
This replacement will have to be done via the operator.

Reply:
Yep. I did that in a branch, but I will do that later. Let's just merge this one first.

Reply:
BTW, we're using the release manifest from midstream here, instead of upstream.
That's why the GCR refs are gone.